vignettes/E_Genomics.Rmd

```diff
@@ -219,22 +222,23 @@ getPooledGeno(x = genoW, type = "count", sex = sexW)[, 1:10]
 (poolW <- getPooledGeno(x = genoW, type = "mean", sex = sexW))[, 1:10]
 ```
 
-Now lets plot and compare the pooled workers to the queen's genotype
+Now lets plot and compare the pooled workers to the queen's genotype (note the
+use of jitter for queen's genotype on the x-axis so we can spread out the dots
+in the plot!).
 
 ```{r plot genoQ with poolW}
- plot( y = poolW, x = genoQ, ylim = c(0, 2), xlim = c(0, 2),
-       ylab = "Average allele dosage in workers",
-       xlab = "Allele dosage in the queen" )
+plot(y = poolW, x = jitter(genoQ), ylim = c(0, 2), xlim = c(0, 2),
```

Comment:
What does this jitter do?

Reply:
Scatters points around the x axis values so we can see the variation a bit better